### DISCUSSION TITLE
Where is this picture in the directory? I cannot find it for the life of me and im trying to change it to my university

### DISCUSSION THREAD

**Discussion by GrahamHNash** [joaomlourenco/novathesis#514] [Q&A]
<img width="608" alt="Screenshot 2024-09-11 at 4 20 15 PM" src="https://github.com/user-attachments/assets/4b4ad72c-7240-44c5-998d-ba4f29f7ec1f">

I cannot find the Nova image logo in the directory at all.

**Comment by joaomlourenco** [comment]
You probably want to design your own cover in LaTeX and then include the "tex" files in the bottom of `Config/4_files.tex` (lines 93 to 95):
```latex
%%------------------------------------------------------------
% User defined cover instead of standard one
%   syntax: \ntaddfile{cover}[1,N,spine]{filename}
% \ntaddfile{cover}[1]{cover-front}  % front cover
% \ntaddfile{cover}[N]{cover-back}   % back cover
% \ntaddfile{cover}[spine]{filename} % book spine
```
